helpdesk_mgmt_project/readme/CONTRIBUTORS.rst

```diff
@@ -13,3 +13,7 @@
 * `Solvos <https://www.solvos.es>`_:
 
   * David Alonso
+
```

Comment:
being a std migration you should do something more to be listed here. Eg: add tests, docs, fix/imp, etc.

Reply:
> being a std migration you should do something more to be listed here. Eg: add tests, docs, fix/imp, etc.

Hi @simahawk ,

Why do you consider that a standard migration is no enough to be considered as a contribution?

IMHO, Manuel did a contribution even if it was a standard migration:

- He checked the code and he tested the module functionally to be sure that there are no issues.
- He opened a new PR (yes, a standard migration, but nobody else did it before).
- Meanwhile, he detected several issues related to other modules and he reported them. He created a couple of [FIX] PR.
- He is open to correct and take in consideration any changes that should be done to the original code in this PR.

So... I come to the conclusion that he is investing his time here. Why shouldn't he be considered as a contributor? Just asking.

THX!

Reply:
@HaraldPanten you are right on but this is a common practice since years.

Now, TBH I remember there was a reference on the [contributing page here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#migrations) but it seems I have a memory leak :sweat_smile: 

Personally I think we could have a sub section or badge specific for migrations in the README to cover such cases.
FTR I've done many migrations myself w/out claiming any authorship in case of "std migrations" :wink: 

Dear @pedrobaeza do you remember / have any reference to any official statement from the past or it was just based on our good practice?

Reply:
Hi, IMO migrations shouldn't be an isolated section, as it depends on each version/area/goal... what is the scope of the migration work. There are some migrations that requires to totally redo the module for fitting the feature in new version. Even if no code changes, the migration requires to check if everything works OK in the new version, so for me this contribution has the same value as adding a field in the existing module and should be included in the same contributors part.

Reply:
Makes sense as I said. Frankly I remember you were against this but that's another story :sweat_smile:
I'll drop a note on the official docs. 

@pedrobaeza tnx for the answer.
@HaraldPanten tnx for challenging.
@manuelregidor tnx for your contrib!

Can you just answer to https://github.com/OCA/helpdesk/pull/227#discussion_r632052681 ? (not blocking)